common/src/misc/logInStrategies/logIn.strategy.ts

```diff
@@ -44,7 +45,10 @@ export abstract class LogInStrategy {
     credentials: ApiLogInCredentials | PasswordLogInCredentials | SsoLogInCredentials
   ): Promise<AuthResult>;
 
-  async logInTwoFactor(twoFactor: TokenRequestTwoFactor): Promise<AuthResult> {
+  async logInTwoFactor(
+    twoFactor: TokenRequestTwoFactor,
+    captchaResponse: string
```

Comment:
Great suggestions - I kept the parity by making the base class default to `null` for that value. This keeps everything upstream the same while also handling your concern of not explicitly handling a non-used value. 👍

Reply:
My concern was more that other strategy subclasses (i.e. api or SSO) appear to take a `captchaResponse` argument in this method, and the expectation would therefore be that they handle captcha as well. Instead it's basically an implicit noop, and it's mixing subclass requirements into the base class.

That said, I think this can be removed altogether, see [my other comment](https://github.com/bitwarden/jslib/pull/698#discussion_r818193106). So it may not matter.

Reply:
Let's tidy this up as a fast follow once we know how we're going to handle SSO + Bitwarden 2FA.